File: Java/src/test/java/org/samples/Part.java
package org.samples;

public class Part {
    private String text;
    private String status;

    public Part(String text, String status) {
        this.text = text;
        this.status = status;
    }

    public boolean isUnchanged() {
        return "unchanged".equals(status);
    }

    public boolean isRemoved() {
        return "removed".equals(status);
    }

    public boolean isAdded() {
        return "added".equals(status);
    }

    @Override
    public String toString() {
        return "`" + text + "`[" + status + "]";
    }
}

File: Python/tests/part.py
class Part:
    def __init__(self, text, status):
        self.text = text
        self.status = status

    def is_unchanged(self):
        return self.status == "unchanged"

    def is_removed(self):
        return self.status == "removed"

    def is_added(self):
        return self.status == "added"

    def __str__(self):
        return f"`{self.text}`[{self.status}]"
